nginx/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# a minimal Nginx container including containerbuddy and a simple virtulhost config
+FROM nginx:latest
```

Comment:
The context of that structure is the docker-compose.yml file though. `nginx` in the compose file corresponds to the nginx/ directory and `cloudflare` corresponds to the cloudflare/ directory. Moving nginx off to examples means moving almost the entire repo into the examples folder: the makefile, the docker-compose files, the start script, and the nginx directory. Which leaves a cloudflare directory which doesn't actually work without the example.